data_iterator.go

```diff
@@ -159,17 +166,26 @@ func (d *DataIterator) Run(tables []*TableSchema) {
 		}()
 	}
 
-	i := 0
+	sorter := MaxTableSizeSorter{DataIterator: d}
+	sortedTableData, err := sorter.Sort(tablesWithData)
```

Comment:
I'm hardcoding the type of sorter we use for now and we can pass this in the config in another PR.